Lib/asyncio/__main__.py

```diff
@@ -153,14 +153,16 @@ def interrupt(self) -> None:
         "ps", help="Display a table of all pending tasks in a process"
     )
     ps.add_argument("pid", type=int, help="Process ID to inspect")
+    formats = [fmt.value for fmt in asyncio.tools.TaskTableOutputFormat]
+    ps.add_argument("--format", choices=formats, default="table")
```

Comment:
This will print the :banana:sv option in the argparse help, right? Is that desired? (I don't care about hiding it, but I think it could be very confusing for users, and also may display weirdly if they don't have a proper font installed).